src/OrbitBase/include/OrbitBase/ThreadUtils.h

```diff
@@ -13,10 +13,13 @@ namespace orbit_base {
 #include <stdint.h>
 [[nodiscard]] uint32_t GetCurrentThreadId();
 [[nodiscard]] std::string GetThreadName(uint32_t tid);
+[[nodiscard]] uint32_t GetCurrentProcessId();
 #else
 #include <sys/types.h>
+#include <unistd.h>
```

Comment:
It looks like this include causes build failures... can you move includes outside of namespace declaration, it should fix the build.

Reply:
Done. Thanks so much for pointing out this!